app/interactive_prompt.py

```diff
@@ -171,6 +172,38 @@ def noOp(data):
     def makeCommand(self, cmdLine, view):
         return {}, u'making stuff'
 
+    def openCommand(self, cmdLine, view):
+        """
+        Opens the file under cursor.
+        """
+        penLine = view.textBuffer.lines[view.textBuffer.penRow]
+        wordStart = wordEnd = view.textBuffer.penCol
+
+        # Locate the boundaries of the word under the cursor
+        while wordStart > 0 and penLine[wordStart - 1] != ' ':
```

Comment:
How about looking for ", ', :, and <, > instead of space.
Consider the use case of .cpp include:
#include "foo/bar/blah.h"
#include <thing/stuff.h>

Reply:
Maybe `isalnum`?
https://docs.python.org/3/library/stdtypes.html#str.isalnum

Reply:
That will reject a period/dot so 'hello.h' would be an issue.
```
>>> u"hello.h".isalnum()
False
```
I'm guessing this will need to be done by hand and require ongoing tweaking; or integration with the parser. E.g. maybe the parser could be queried for the path under the cursor. 

That would be an advanced change though. So I wouldn't recommend digging in there unless your really interested in getting deep into it. The code for the parser is a bit weird for performance reasons. 

I'm going to suggest we commit this PR without those tweaks or parser changes. The PR is an advancement over what we had before. Those tweaks or parser changes can happen in future PRs.

(Trivia: I'd intended to prototype the editor in Python and migrate to a systems language when Python became unsuitable (not fast enough), but I keep being able to make Python work so well that the need to change hasn't been needed; though that means the parser is a touch unusual).